src/views/ModelPlan/CRTDL/CRTDLs/__snapshots__/index.test.tsx.snap

```diff
@@ -142,3 +142,142 @@ exports[`Model Plan CR and TDL page > matches snapshot 1`] = `
+            >
+              <a
+                class="usa-breadcrumb__link"
+                href="/models/f11eb129-2c80-4080-9440-439cbe1a286f/task-list"
+              >
+                <span>
+                  Model Plan task list
+                </span>
+              </a>
+            </li>
+            <li
+              aria-current="page"
+              class="usa-breadcrumb__list-item usa-current"
+            >
+              CR and TDLs
+            </li>
+          </ol>
+        </nav>
+        <h1
+          aria-live="polite"
+          class="mint-h1 margin-top-4 margin-bottom-0"
+          tabindex="-1"
+        >
+          CR and TDLs
+        </h1>
+        <p
+          class="margin-top-0 margin-bottom-2 font-body-lg"
+          data-testid="model-plan-name"
+        >
+          for My Plan
+        </p>
+        <p
+          class="margin-bottom-2 font-body-md line-height-body-4"
+        >
+          Add, manage, and update any known Change Requests (CRs) and Technical Direction Letters (TDLs) associated with your model. More information about each CR or TDL can be found in Enterprise Electronic Change Information Management Portal (ECHIMP). If you need access to ECHIMP, please contact the MINT Team at 
+          <a
+            class="usa-link"
+            href="[email]@cms.hhs.gov"
+          >
+            MINTTeam@cms.hhs.gov.
+          </a>
+        </p>
+        <a
+          class="usa-link display-inline-flex flex-align-center margin-y-3"
+          href="/models/f11eb129-2c80-4080-9440-439cbe1a286f/task-list"
+        >
+          <svg
+            aria-hidden="true"
+            class="usa-icon margin-right-1"
+            focusable="false"
+            height="1em"
+            role="img"
+            viewBox="0 0 24 24"
+            width="1em"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
+            />
+          </svg>
+          Return to Model Plan task list
+        </a>
+        <h4
+          class="margin-top-2 margin-bottom-1"
+        >
+          CR and TDLs
+        </h4>
+        <a
+          class="usa-button"
+          href="/models/f11eb129-2c80-4080-9440-439cbe1a286f/cr-and-tdl/add-cr-and-tdl"
+        >
+          Add a CR or TDL
+        </a>
+        <div
+          class="usa-alert usa-alert--error margin-top-3"
+          data-testid="formik-validation-errors"
+          role="alert"
+        >
+          <div
+            class="usa-alert__body"
+          >
+            <h3
+              class="usa-alert__heading"
+              tabindex="-1"
+            >
+              There is a problem
+            </h3>
+            <button
+              class="usa-error-message usa-alert__text mint-error-alert__message text-left"
+              type="button"
+            >
+              Failed to fetch Model Plan CRs and TDLs
+            </button>
+          </div>
```

Comment:
These snapshots are duplicating and failing to render the data.  It doesnt seem too related to this PR and hopefully addressed in the unit test PR.  I would wait for the other PR to be merged and recheck if this get updated correctly